finite_state_sdk/queries.py

```diff
@@ -247,11 +247,25 @@ def artifact_variables(artifact_id=None, business_unit_id=None):
                     id
                     name
                     createdAt
+                    createdBy {
+                        id
+                        email
+                        __typename
+                    }
+                    deletedAt
```

Comment:
Just to be sure, what do you expect the customer would use the deletedAt field for? With the status work we are going way from soft deletes